ui/src/components/song/SongCard.tsx

```diff
@@ -41,88 +43,104 @@ const SongCard = (props: SongCardProps) => {
+    <>
+      <Container
+        onClick={() => navigate(`/song/${_id}`)}
+        sx={{
+          borderRadius: '8px',
+          border: 1,
+          borderColor: '#49454F',
+          backgroundColor: 'primary.darkest',
+          py: 2,
+        }}
+      >
+        <Stack direction="row" display="flex" justifyContent="space-between">
+          <Stack>
+            <Typography variant="h2" color={'secondary.main'}>
+              {title}
+            </Typography>
+            <Typography variant="subtitle1" color={'secondary.main'}>
+              {artist}
+            </Typography>
+          </Stack>
+          <>
+            <Box sx={{ height: '30px', width: '30px' }}>
+              <VisibilityIcon
+                onClick={(event) => {
+                  event.stopPropagation();
+                  modalOpen ? handleClose() : handleOpen();
+                }}
+                // onMouseLeave={handleClose}
+                sx={{ color: 'secondary.main' }}
+              />
+            </Box>
+          </>
+        </Stack>
+        <Stack direction={isDesktop ? 'row' : 'column'} spacing={isDesktop ? 4 : 1}>
+          {CardFields &&
+            showDetails !== false &&
+            CardFields.map((field, i) => {
+              return (
+                <Stack direction="row" display="flex" key={i} spacing={1}>
+                  {displayData[i] ? (
+                    <>
+                      <Typography
+                        variant="body2"
+                        color="#9E9E9E"
+                        width={isDesktop ? '100%' : '40%'}
+                      >
+                        {field}
+                      </Typography>
+                      {Array.isArray(fieldData[i]) ? (
+                        <SongFieldArray data={fieldData[i]} />
+                      ) : (
+                        <Typography variant="body2" color={'#CCC2DC'}>
+                          {fieldData[i] ?? '-'}
+                        </Typography>
+                      )}
+                    </>
+                  ) : null}
+                </Stack>
+              );
+            })}
+        </Stack>
+      </Container>
+      <Modal open={modalOpen} onClose={handleClose}>
+        <Stack
+          sx={{
+            position: 'absolute',
+            top: '50%',
+            left: '50%',
+            transform: 'translate(-50%, -50%)',
+            width: 400,
+            bgcolor: '#171717',
+            borderRadius: 8,
+            boxShadow: 2,
+            p: 4,
+          }}
+          spacing={1}
+        >
+          <Typography variant="h2">{title}</Typography>
+          <Typography variant="subtitle1" sx={{ color: 'secondary.light' }}>
             {artist}
           </Typography>
+          <Typography sx={{ color: 'secondary.light' }}>
+            {lyricsPreview && !lyricsPreview.includes('[')
+              ? lyricsPreview
+              : getLyricsPreview(chordLyrics)}
```

Comment:
what is this chordLyrics yaa? I feel like we don't need to have the option to have chords in lyric preview hehe